en/docs/design/api-security/api-authentication/disable-security.md

```diff
@@ -0,0 +1,19 @@
+# Disabling Security for APIs
```

Comment:
Shall we add some introduction for this scenario?

Reply:
Fixed with https://github.com/wso2/docs-apim/pull/5555/commits/fef17550bbb360bbf5e86e06bf21b021bcca9ff8